src/api/Users.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @flow
+ * @file Helper for the box Users API
+ * @author Box
+ */
+
+import Base from './Base';
+
+/**
+ * Fields we want include:
+ *  -
+ */
+
+class Users extends Base {
+    /**
+     * API URL for Users
+     *
+     * @param {string} [id] - A box user id. Defaults to 'me' if empty.
+     * @return {string} base url for users
+     */
+    getUrl(id?: string): string {
+        const userId = id || 'me';
+        return `${this.getBaseApiUrl()}/users/${userId}?fields=login,name`;
```

Comment:
Fields should be addable to the API call directly now. e.g https://github.com/box/box-ui-elements/commit/df2069cf64a4fac9457e5778547373d2fe42d801#diff-b7fcdd8925731f98f2c2f5413e508d6aR542

Reply:
fields= needed?
It should get that info without fields specified

Reply:
You are correct! We don't need fields for this call!